Resources/Prototypes/Entities/Objects/Devices/Circuitboards/Machine/production.yml

```diff
@@ -312,3 +312,17 @@
           Amount: 1
           DefaultPrototype: Beaker
           ExamineName: Glass Beaker
+
+- type: entity
+  id: StasisBedMachineCircuitboard
+  parent: BaseMachineCircuitboard
+  name: Stasis Bed (Machine Board)
+  components:
+    - type: MachineBoard
+      prototype: StasisBed
+      requirements:
+        Capacitor: 1
+        Manipulator: 1
+      materialRequirements:
+        Cable: 3
```

Comment:
as said above this needs to be much more expensive, ik the board already requires gold but still a bit more